include/nuttx/sched_note.h

```diff
@@ -348,12 +379,13 @@ void sched_note_irqhandler(int irq, FAR void *handler, bool enter);
 #  define sched_note_irqhandler(i,h,e)
 #endif
 
+void sched_note_message(FAR const char *fmt, ...);
```

Comment:
It's better to move sched_note_message related change to another patch, so we can merge it to mainline directly. Also please don't forget to update the apps/system/sched_note/.

Reply:
I got. I'll remove sched_note_message features to make another PR for it.
Also ok for the changes of syscall arg/result and irq handler.